apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,56 @@
 import { Component, OnInit } from '@angular/core';
+import { CityStore } from '../../data-access/city.store';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { City } from '../../model/city.model';
+import { CardComponent } from '../../ui/card/card.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <app-card
+      [list]="cities"
+      class="bg-light-blue"
+      (deleteItem)="deleteCity($event)">
+      <img image src="assets/img/city.png" alt="City" width="200px" />
+      <button
+        addButton
+        class="rounded-sm border border-blue-500 bg-blue-300 p-2"
+        (click)="addNewCity()">
+        Add
+      </button>
+    </app-card>
+  `,
   standalone: true,
-  imports: [],
+  styles: [
+    `
+      .bg-light-blue {
+        background-color: rgba(0, 0, 250, 0.1);
+      }
+    `,
+  ],
+  imports: [CardComponent],
 })
 export class CityCardComponent implements OnInit {
-  constructor() {}
+  cities: City[] = [];
 
-  ngOnInit(): void {}
+  constructor(
+    private http: FakeHttpService,
+    private store: CityStore,
+  ) {}
+
+  ngOnInit(): void {
+    this.http.fetchCities$.subscribe((s) => this.store.addAll(s));
+    this.store.cities$.subscribe((s) => (this.cities = s));
```

Comment:
prefer declarative coding and not imperative coding.

You can use signal as well.